app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarFragment.kt

```diff
@@ -514,6 +552,59 @@ class CalendarFragment :
             .show()
     }
 
+    private fun setMonthView() {
+        monthYearText.text = monthYearFromDate(selectedDate)
+        val daysInMonth: ArrayList<String> = daysInMonthArray(selectedDate)
+
+        val eventMap = calendarController.getEventsForMonth(selectedDate)
+
+        if (!::monthViewAdapter.isInitialized) {
+            monthViewAdapter = MonthViewAdapter(daysInMonth, eventMap)
+            monthRecyclerView.adapter = monthViewAdapter
+        } else {
+            monthViewAdapter.updateData(daysInMonth, eventMap)
+        }
+
+        val layoutManager: RecyclerView.LayoutManager = GridLayoutManager(requireContext(), 7)
+        monthRecyclerView.layoutManager = layoutManager
+    }
+
+    private fun daysInMonthArray(date: LocalDate): ArrayList<String> {
```

Comment:
I dont like appending types to names (see https://youtu.be/-J3wNP6u5YU?t=126 for a more in depth oppinion)

Reply:
```suggestion
    private fun daysInMonth(date: LocalDate): ArrayList<String> {
```